DCode.Web/Scripts/App_JS/Controllers/Requestor.js

```diff
@@ -280,16 +280,14 @@
         }
 
 
-        $scope.reviewApplicant = function (task, applicant, approvalApplicantId, wrkAgainValue, rating, comments) {
+        $scope.reviewApplicant = function (task, applicant, approvalApplicantId) {
             $http({
                 method: 'POST',
                 url: '/Requestor/ReviewTask',
                 data: {
                     TaskId: task.Id,
                     ApplicantId: applicant.ApplicantId,
-                    ApprovedApplicantId: approvalApplicantId,
-                    Rating: rating,
-                    WorkAgain: wrkAgainValue

```

Comment:
We can remove the three parameters in the ReviewTask Action if that is this is the only place where it is being called from.